.github/workflows/scripts/check_ignore.py

```
@@ -0,0 +1,265 @@
+        None
+
+    Returns:
+        None
+
+    """
+    print(
+        subprocess.check_output(["git", "branch"]).decode("utf-8").strip("\n")
+    )
+
+    current_branch = (
+        subprocess.check_output(["git", "branch", "--show-current"])
```

Ruff (0.14.7)

Starting a process with a partial executable path

(S607)